recipe/meta.yaml

```
@@ -16,16 +16,16 @@ build:
 
 requirements:
   build:
+    - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - cmake
     - make
```

```suggestion
    - {{ stdlib("c") }}
    - {{ compiler('cxx') }}
    - cmake
    - make
host:
    - zlib
```

I realized I can suggest the change that I asked for.

---

Sorry needs to be indented properly.